src/test/index.js

```diff
@@ -2,34 +2,45 @@
 
 const pmap = require('p-map')
 const _ = require('lodash')
+const path = require('path')
 
 const node = require('./node')
 const browser = require('./browser')
 
 const userConfig = require('../config/user')
 
+const taskEnabled = (ctx, name) => _.includes(ctx.target, name)
+
 const TASKS = [{
+  id: 'node',
   title: 'Test Node.js',
-  task: node,
-  enabled: (ctx) => _.includes(ctx.target, 'node')
```

Comment:
Lets's keep these here, this is the same structure of tasks we use for listr in other places.